vyper/compiler/output.py

```diff
@@ -23,6 +23,14 @@ def build_ast_dict(compiler_data: CompilerData) -> dict:
     return ast_dict
```

Comment:
i think now `build_ast_dict` is dead code? can we just replace the old `build_ast_dict` body with the new `build_unfolded_ast_dict` body?